include/votca/tools/calculator.h

```diff
@@ -77,20 +77,40 @@ class Calculator {
    * @param output stream
    */
   void DisplayOptions(std::ostream &out);
+
+  /**
+   * \brief Loads default options stored in VOTCASHARE
+   */
+  void LoadDefaults(std::string package = "tools");
+
   /**
    * \brief Updates options with default options stored in VOTCASHARE
    *
    * If a value is not given or tag is not present and at the same time
    * a default value exists in the corresponding XML file in VOTCASHARE
    * a tag is created and/or a default value is assigned to it
    */
-  void UpdateWithDefaults(Property &options, std::string package = "tools");
+  void UpdateWithUserOptions(const Property &options);
```

Comment:
which options is the input ? The user or the default?

Reply:
I renamed it in the .cc to `user_options` and I am going to do the same for the definition